### PR TITLE
Constants

### PR DESCRIPTION
**Pull request by ShrutiMandre**
Created constants with the port numbers set to 0 temporarily.

### CODE REVIEW

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -16,4 +16,70 @@
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
+
+    // ADD THE PORT NUMBERS TO CONSTANTS!!!!!
+
+    public static final class DriveConstants {
+        public static final int kDriveMotor1 = 0;
```
Comment (zbross) [inline src/main/java/frc/robot/Constants.java:23]: Change these to kLeftDriveMotor1Port, etc.  We need to know which side is which.

File: src/main/java/frc/robot/Constants.java [hunk 2]
```diff
@@ -16,4 +16,70 @@
...
+        public static final int kDriveMotor2 = 0;
+        public static final int kDriveMotor3 = 0;
+        public static final int kDriveMotor4 = 0;
+
+        public static final int kLeftTalonPort = 0;
```
Comment (zbross) [inline src/main/java/frc/robot/Constants.java:28]: The left and right talons don't exist for the drivetrain this year.

File: src/main/java/frc/robot/Constants.java [hunk 3]
```diff
@@ -16,4 +16,70 @@
...
+        public static final int kRightTalonPort = 0;
+        
+        public static final int kGyroSensor = 0; //IMU for point turn
```
Comment (zbross) [inline src/main/java/frc/robot/Constants.java:31]: The gyro sensor doesn't need a port number.

File: src/main/java/frc/robot/Constants.java [hunk 4]
```diff
@@ -16,4 +16,70 @@
...
+        public static final double kWheelDiameter = 0; //ADD UNIT OF MEASUREMENT
+
+    }
+
+    public static final class LauncherConstants {
+        public static final int kLauncherMotor1 = 0;
+        public static final int kLauncherMotor2 = 0;
+
+        // pistons adjust the angle of the launcher
+        public static final int kLauncherPiston1 = 0; // double solenoid
```
Comment (zbross) [inline src/main/java/frc/robot/Constants.java:42]: There's only one solenoid, but it's driving two pistons.  Change these to kLauncherSolenoid or kLauncherPistons.

File: src/main/java/frc/robot/Constants.java [hunk 5]
```diff
@@ -16,4 +16,70 @@
...
+        public static final int kLauncherPiston2 = 0; // double solenoid
+
+    }
+
+    public static final class IntakeConstants {
+        
+        // beam breaker detects the number of powercells in the cellevator
+        public static final int kBeamBreak1 = 0;
```
Comment (zbross) [inline src/main/java/frc/robot/Constants.java:50]: There are two ports per beam break.

File: src/main/java/frc/robot/Constants.java [hunk 6]
```diff
@@ -16,4 +16,70 @@
...
+        public static final int kBeamBreak2 = 0;
+
+        public static final int kIntakeMotor1 = 0;
```
Comment (zbross) [inline src/main/java/frc/robot/Constants.java:53]: There's only one intake motor.

File: src/main/java/frc/robot/Constants.java [hunk 7]
```diff
@@ -16,4 +16,70 @@
...
+        public static final int kIntakeMotor2 = 0;
+        public static final int kIntakeMotor3 = 0;
+        public static final int kIntakeMotor4 = 0;
+        public static final int kIntakeMotor5 = 0;
+
+        // pistons that lower and raise the intake
+        public static final int kIntakePiston1 = 0; // double solenoid
```
Comment (zbross) [inline src/main/java/frc/robot/Constants.java:60]: There's only one solenoid for the intake pistons.

File: src/main/java/frc/robot/Constants.java [hunk 8]
```diff
@@ -16,4 +16,70 @@
...
+    public static final class HangerConstants {
+
+        public static final int kHangerMotor1 = 0;
+        public static final int kHangerMotor2 = 0;
+
+    }
+
+    public static final class LimeLightConstants {
+
+        public static final int kLimelight = 0; // used for vision
```
Comment (zbross) [inline src/main/java/frc/robot/Constants.java:81]: The limelight doesn't have a port number.

**Review by zbross — APPROVED** [review]
Please update the constants according to my comments.  Make a new branch to fix these, but I'm approving the changes for now so that you can begin the drivetrain.

File: src/main/java/frc/robot/Constants.java (hunk 5 above)
Comment (irinale11) [reply src/main/java/frc/robot/Constants.java:50]: We checked, and there are two ports but we only need to put the output port in the code.

**Pull request merged**